Polyhedron/demo/Polyhedron/Scene_polyhedron_item.cpp

```diff
@@ -359,18 +334,18 @@ Scene_polyhedron_item::initialize_buffers(CGAL::Three::Viewer_interface* viewer)
         }
         vaos[0]->release();
         //gouraud
-        vaos[4]->bind();
+        vaos[2]->bind();
         buffers[0].bind();
         program->enableAttributeArray("vertex");
         program->setAttributeBuffer("vertex",GL_FLOAT,0,4);
         buffers[0].release();
 
```

Comment:
Can't you use the enum trick here too?
What about `voas`, is it relevant to have something similar instead of hard coded integer?